tests/scripts/config.source

```diff
@@ -1,26 +1,5 @@
-# Provide values from config
-settings="$(dirname "$(realpath "$0")")/config/pulp/settings.toml"
-PULP_BASE_URL="$(sed -n -e 's/^base_url\s*=\s*"\(\S*\)"\s*$/\1/p' "$settings")"
-VERIFY_SSL="$(sed -n -e 's/^verify_ssl\s*=\s*\(\S*\)\s*$/\1/p' "$settings")"
-
-# Configure path to config
-export XDG_CONFIG_HOME="$(dirname "$(realpath "$0")")/config"
-
-# Constants used in tests
-PULPCORE_VERSION=$(pulp status | jq -r '.versions | .[] | select(.component=="pulpcore").version')
-PULP_FIXTURES_URL="${PULP_FIXTURES_URL:-https://fixtures.pulpproject.org}"
-FILE_REMOTE_URL="${PULP_FIXTURES_URL}/file/PULP_MANIFEST"
-CONTAINER_REMOTE_URL="https://registry-1.docker.io"
-CONTAINER_IMAGE="hello-world"
-RPM_REMOTE_URL="${PULP_FIXTURES_URL}/rpm-unsigned"
-ANSIBLE_COLLECTION_REMOTE_URL="https://galaxy.ansible.com/"
-ANSIBLE_ROLE_REMOTE_URL="https://galaxy.ansible.com/api/v1/roles/?namespace__name=elastic"
-
 # Library for test helper functions
 
-TMP="$(mktemp -d)"
-trap "rm -rf $TMP" EXIT
```

Comment:
Somehow this trap was never executed. Maybe because we set a new trap in every other test script.